src/AspNetCore/Src/applicationinfo.cpp

```diff
@@ -563,6 +564,11 @@ APPLICATION_INFO::RecycleApplication()
                 g_pHttpServer->RecycleProcess(L"On Demand by AspNetCore Module for recycle application failure");
             }
         }
+        else 
```

Comment:
Use CThread? :)

Reply:
will Cthread  be heavy due to constructor and destructor?

Reply:
Yes

Reply:
I am fine with using Cthread on any non-request code path. Recycling the application is per application, and I prefer the benefits of using CThread s.t. we don't need to close the handle.

Reply:
I really want to do some performance test to see the perf impact before using standard libraries. NGinx is purely based on C and has much better performance.

Reply:
@pakrym please open another working issue so that we can switch to CTHREAD cross the projects. I don't want partially switch to CThread in this change.